docs/pre-commit.md

```diff
@@ -0,0 +1,21 @@
+# pre-commit
+
+To use `typos` with [`pre-commit`](https://pre-commit.com), point its
+config at this repository:
+
+```yaml
+repos:
+  - repo: https://github.com/crate-ci/typos
+    rev: v1.0.6
```

Comment:
You can update `Cargo.toml`'s
```
[package.metadata.release]
pre-release-replacements = [
```
To get this updated on every release automatically.

Reply:
(these changes can be validated with `cargo release --dry-run major -vvv`)

Reply:
Thanks for the tip, rust/cargo newbie here. Added but wasn't able to test, hopefully I got it even remotely right.

```shellsession
$ cargo release --dry-run major -vvv
error: no such subcommand: `release`
$ cargo --version
cargo 1.52.0 (69767412a 2021-04-21)
```
...ah, need to additionally install "cargo-release" for that to work. Did that, and it appears to be doing the right thing.